public/video-ui/src/components/Pluto/PlutoProjectPicker.js

```diff
@@ -0,0 +1,94 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { ManagedForm, ManagedField } from '../ManagedForm';
+import SelectBox from '../FormFields/SelectBox';
+
+class PlutoProjectPicker extends React.Component {
+  static propTypes = {
+    video: PropTypes.object.isRequired,
+    saveVideo: PropTypes.func.isRequired
+  };
+
+  hasPlutoCommissions = () => this.props.pluto && this.props.pluto.commissions.length !== 0;
+
+  componentWillMount() {
+    if (!this.hasPlutoCommissions()) {
+      this.props.plutoActions.getCommissions().then(() => {
+        const {plutoData} = this.props.video;
+        if (plutoData.commissionId) {
+          this.props.plutoActions.getProjects(plutoData);
+        }
+      });
+    }
+  }
+
+  onCommissionSelection() {
+    const {plutoData} = this.props.video;
+    this.props.plutoActions.getProjects(plutoData);
+  }
+
+  render() {
+    const {video, saveVideo} = this.props;
+    const {commissions, projects} = this.props.pluto;
+
+    return (
+      <div>
+        <div className="video__detailbox__header__container">
+          <header className="video__detailbox__header">
+            Pluto
+          </header>
+        </div>
+        <div className="form__group">
+          <ManagedForm
+            data={video}
+            updateData={saveVideo}
+            editable={true}
+            formName="Pluto"
+          >
+            <ManagedField
+              fieldLocation="plutoData.commissionId"
+              name="Commission"
+              isRequired={false}
+              updateSideEffects={() => this.onCommissionSelection()}
```

Comment:
Why is the call to `onCommissionSelection` wrapped inside the function?

Reply:
I need access to `this.props.video`